templates/adding_a_new_model/cookiecutter-template-{{cookiecutter.modelname}}/modeling_tf_{{cookiecutter.lowercase_modelname}}.py

```diff
@@ -216,8 +215,8 @@ def call(
         elif past_key_value is not None:
             key_layer = self.transpose_for_scores(self.key(inputs=hidden_states), batch_size)
             value_layer = self.transpose_for_scores(self.value(inputs=hidden_states), batch_size)
-            key_layer = tf.concatenate([past_key_value[0], key_layer], dim=2)
```

Comment:
Wow great mistranslation from PyTorch :sweat_smile:

Reply:
I am guilty of this (and if I added the tests at that time, I could detect it ...)